app/models/concerns/auction_scopes.rb

```diff
@@ -3,6 +3,7 @@ module AuctionScopes
 
   included do
     scope :accepted, -> { where(status: statuses['accepted']) }
+    scope :accepted_pending_payment_url, -> { where(status: statuses['accepted_pending_payment_url']) }
     scope :delivery_accepted, -> { where(status: statuses['accepted']).where.not(accepted_at: nil) }
```

Comment:
This scope is slightly different now. It seems like before it meant ("delivery accepted and we have a payment URL") and now the second clause is redundant. I would look at if this is still used and we might need to modify name/definition